app/controllers/observation_matrices_controller.rb

```diff
@@ -310,4 +357,105 @@ def observation_matrix_params
     params.require(:observation_matrix).permit(:name)
   end
 
-end
+  def nexus_import_options_params
+    boolean_options = [:Match_otu_to_taxonomy_name, :Match_otu_to_name,
```

Comment:
@kleintom I think these can be lowercase symbols and in views/rendering you can do ` a.to_s.humanize`